.github/workflows/release-js.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     tags:
       - v*
+  workflow_dispatch:
```

Comment:
This option will add a support for manually rerunning workflows. Useful when Sonatype has an issue